packages/legend-studio-manual-tests/src/__tests__/roundtrip-grammar/RoundtripGrammar.test.ts

```diff
@@ -128,6 +128,23 @@ const checkGrammarRoundtrip = async (
         change.oldPath !== '__internal__::SectionIndex',
     ).length,
   ).toBe(0);
+
+  // compose grammar and compare that with original grammar text
+  // NOTE: this is optional test as `grammar text <-> protocol` test should be covered
+  // in engine already.
+  // Here, we do it just so we might be able to detect problem in the grammar roundtrip in engine
+  const transformJsonToGrammarResult = await axios.post(
```

Comment:
This is under the assumption the grammar is formatted in the correct order right?

Reply:
yeah, a bit strict, but I thought we might as well to cover more cases